README.md

```diff
@@ -42,25 +40,13 @@ Here's an example ESLint configuration that:
 - Enables the `recommended` configuration
 - Enables an optional/non-recommended rule
 
-Note: you might need to set `sourceType` to `script` (most users) (use `module` for ESM/TypeScript).
-
-### <a name='eslintrc'></a>**[.eslintrc.json](https://eslint.org/docs/latest/use/configure/configuration-files)**
-
-```json
-{
-  "extends": ["plugin:eslint-plugin/recommended"],
-  "rules": {
-    "eslint-plugin/require-meta-docs-description": "error"
-  }
-}
-```
-
-### <a name='flat'></a>[`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files-new) (requires eslint>=v8.23.0)
+Note: you might need to set `sourceType` to `module` or `script` depending on your codebase.
 
 ```js
```

Comment:
For clarity, it would be nice to include the filename in at least one of these examples to people don't try to use it with the old eslintrc format:

```suggestion
```js
// eslint.config.js
```